implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -67,6 +67,10 @@ pub struct CreateCommand {
     /// Time to wait before retrying to connect to outlet.
     #[arg(long, display_order = 900, id = "RETRY", default_value = "20s", value_parser = duration_parser)]
     retry_wait: Duration,
+
+    /// Override default timeout
+    #[arg(long, value_parser = duration_parser)]
+    timeout: Option<Duration>,
```

Comment:
What if we also specify the default by referencing the node manager default timeout?

Reply:
This might not be straightforward to fix, I'll address it in  separate PR.